Jenkinsfile_CNP

```diff
@@ -63,9 +66,11 @@ withPipeline(type, product, component) {
 
     onMaster() {
         enablePactAs([AppPipelineDsl.PactRoles.CONSUMER])
+        enablePactAs([AppPipelineDsl.PactRoles.PROVIDER])
     }
 
     onPR() {
         enablePactAs([AppPipelineDsl.PactRoles.CONSUMER])
+        enablePactAs([AppPipelineDsl.PactRoles.PROVIDER])
```

Comment:
we probably don't want to enable provider tests on or level as we only want to be verifying with what's on master not experimental branches :)